inference-engine/src/plugin_api/ie_icore.hpp

```diff
@@ -123,6 +123,15 @@ class ICore {
      */
     virtual std::vector<std::string> GetAvailableDevices() const = 0;
 
+    /**
+     * @brief Checks whether device supports Export & Import functionality of network
+     *
+     * @param deviceName - A name of a device to get a metric value.
+     * @return True if device has IMPORT_EXPORT_SUPPORT metric in SUPPORTED_METRICS and
+     * this metric returns 'true', False otherwise.
+     */
+    virtual bool DeviceSupportsImportExport(const std::string& deviceName) const = 0;
```

Comment:
shall we generalize function to check any given metric? it is overkill to have a full-blown function to check the single metric, imho

Reply:
also to separate the Import from Export, e.g. MULTI presumbaly doesn't care about Export, only Import

Reply:
There should be no device which supports export (write network to blob file), but doesn't support 'import'. And vice versa, device/plugin can't support 'import from blob' which it can't export before. That's why metric is called IMPORT_EXPORT_SUPPORT.
Metric in general can return any type, not just bool. So, it will probably not be applicable for any metric. Also this function doesn't only checks metric presence in 'SUPPORTED_METRICS', but also gets its value. So, this logic was used in several places and I moved it here to avoid code duplication. If needed I can move it out of here, and define same helper function (with DeviceSupportsCaching name) in MULTI plugin